godot-macros/src/docs.rs

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+use crate::class::{ConstDefinition, Field, FuncDefinition, SignalDefinition};
+mod md2bb;
+use proc_macro2::{Ident, TokenStream};
+use quote::ToTokens;
+use venial::*;
+
+pub fn define(base: String, description: Vec<Attribute>, members: Vec<Field>) -> TokenStream {
+    let desc = docs(description);
+    let members = members
+        .into_iter()
+        .filter(|x| x.var.is_some() | x.export.is_some())
+        .map(member)
+        .collect::<String>();
+    quote::quote![[#base, #desc, #members]]
+}
+
+pub fn inherent_impl(
+    functions: impl IntoIterator<Item = FuncDefinition>,
+    constants: impl IntoIterator<Item = ConstDefinition>,
+    signals: impl IntoIterator<Item = SignalDefinition>,
+) -> TokenStream {
+    let methods = functions.into_iter().map(method).collect::<String>();
+    let signals = signals.into_iter().map(signal).collect::<String>();
+    let constants = constants
+        .into_iter()
+        .map(|ConstDefinition { raw_constant: x }| x)
+        .map(constant)
+        .collect::<String>();
+    quote::quote![[#methods, #signals, #constants]]
```

Comment:
```suggestion
    quote::quote! { [#methods, #signals, #constants] }
```
in gdext, `quote!` should always be called like this